observability/micrometer/src/test/java/org/wildfly/extension/micrometer/SubsystemParsingTestCase.java

```diff
@@ -45,7 +45,7 @@ public SubsystemParsingTestCase(MicrometerSubsystemSchema schema) {
         super(MicrometerExtension.SUBSYSTEM_NAME, new MicrometerExtension());
         this.schema = schema;
         this.testXml = "<subsystem xmlns=\"" + this.schema.getNamespace() +
-                "\" endpoint=\"http://localhost:4318/v1/metrics\"></subsystem>";
+                "\"><otlp-registry endpoint=\"http://localhost:4318/v1/metrics\"/></subsystem>";
```

Comment:
This is a pre-existing flaw, but this test case needs beefing up to ensure proper testing of the basic management model behavior.

1) There should be coverage of a test xml that includes all attributes. The exposed-subsystems attribute is not covered. (Note that the this.testXml var here can be beefed up and used; you don't need another xml chunk.)
2) For any attribute that supports expressions, the xml value should be an expression of the form '${test.<attribute_name>:<adefaultvalue>}'.  The exact name of the expression property doesn't matter; the idea is to include an expression, but with a default value so the test isn't required to actually set the property.